SPECS/libiothsm-std/libiothsm-std.spec

```diff
@@ -1,6 +1,6 @@
 Summary:        Azure IoT standard mode HSM lib
 Name:           libiothsm-std
-Version:        1.1.0
+Version:        1.1.2
```

Comment:
If these files need to always have the same version then let's also add them into the entangled spec inside `.github/workflows/check_entangled_specs.py`.

Reply:
A bit of explanation: the sets inside the `version_matching_groups` list is for specs, which must have the same version but releases may vary as opposed to the sets inside the `version_release_matching_groups` list, where both the version and the release must be identical.

Reply:
Good idea, I've updated this file